client/state/data-layer/wpcom/sites/atomic/transfer/index.js

```diff
@@ -0,0 +1,163 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ATOMIC_TRANSFER_INITIATE,
+	ATOMIC_TRANSFER_REQUEST,
+	ATOMIC_TRANSFERS_REQUEST,
+} from 'state/action-types';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { requestSite } from 'state/sites/actions';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import {
+	fetchAtomicTransfer,
+	setAtomicTransfer,
+	atomicTransferComplete,
+} from 'state/atomic-transfer/actions';
+import { COMPLETED, ERROR } from 'state/atomic-transfer/constants';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+import { pluginUploadError, updatePluginUploadProgress } from 'state/plugins/upload/actions';
+import { errorNotice } from 'state/notices/actions';
+
+/**
+ * Initiates a site transfer to Atomic.
+ *
+ * @param {?Object} action default action to call on HTTP events
+ * @returns {Object} action object
+ */
+export const initiateTransfer = action => {
+	const { siteId, module } = action;
+
+	return http(
+		{
+			apiNamespace: 'wpcom/v2',
+			method: 'POST',
+			path: `/sites/${ siteId }/atomic/transfers`,
+			query: {
+				plugin_zip: module.pluginZip,
+				plugin_slug: module.pluginSlug,
+				theme_zip: module.themeZip,
+			},
+		},
+		action
+	);
+};
+
+/**
+ * Retreives a list of Atomic transfer records.
+ *
+ * @param {?Object} action default action to call on HTTP events
+ * @returns {Object} action object
+ */
+export const getTransfers = action =>
+	http(
+		{
+			apiNamespace: 'wpcom/v2',
+			method: 'GET',
+			path: `/sites/${ action.siteId }/atomic/transfers`,
+		},
+		action
+	);
+
+/**
+ * Retreives a specific Atomic transfer record.
+ *
+ * @param {?Object} action default action to call on HTTP events
+ * @returns {Object} action object
+ */
+export const getTransfer = action =>
+	http(
+		{
+			apiNamespace: 'wpcom/v2',
+			method: 'GET',
+			path: `/sites/${ action.siteId }/atomic/transfers/${ action.transferId }`,
+		},
+		action
+	);
+
+const showErrorNotice = error => {
+	if ( error.error === 'invalid_input' ) {
+		return errorNotice( translate( 'The uploaded file is not a valid zip.' ) );
+	}
+
+	if ( error.error === 'api_success_false' ) {
+		return errorNotice( translate( 'The uploaded file is not a valid plugin or theme.' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'The uploaded file is not a valid plugin.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).